### ISSUE TITLE
Training Starts, Stops.

### ISSUE THREAD

**Issue by taellinglin** [Mangio621/Mangio-RVC-Fork#171]
As soon as it starts to train it stops and says I pressed stop but I didn't!

2023-12-01 08:46:56,399	SLILY	INFO	{'train': {'log_interval': 200, 'seed': 1234, 'epochs': 20000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 12, 'fp16_run': True, 'lr_decay': 0.999875, 'segment_size': 12800, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'max_wav_value': 32768.0, 'sampling_rate': 40000, 'filter_length': 2048, 'hop_length': 400, 'win_length': 2048, 'n_mel_channels': 125, 'mel_fmin': 0.0, 'mel_fmax': None, 'training_files': './logs\\SLILY/filelist.txt'}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [10, 10, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'use_spectral_norm': False, 'gin_channels': 256, 'spk_embed_dim': 109}, 'model_dir': './logs\\SLILY', 'experiment_dir': './logs\\SLILY', 'save_every_epoch': 5, 'name': 'SLILY', 'total_epoch': 20, 'pretrainG': 'pretrained_v2/f0G40k.pth', 'pretrainD': 'pretrained_v2/f0D40k.pth', 'version': 'v2', 'gpus': '0', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 1, 'save_every_weights': '1', 'if_cache_data_in_gpu': 0}
2023-12-01 08:46:57,809	SLILY	INFO	loaded pretrained pretrained_v2/f0G40k.pth
2023-12-01 08:46:58,021	SLILY	INFO	loaded pretrained pretrained_v2/f0D40k.pth
2023-12-01 08:47:15,728	SLILY	INFO	Train Epoch: 1 [0%]
2023-12-01 08:47:15,730	SLILY	INFO	[0, 0.0001]
2023-12-01 08:47:15,730	SLILY	INFO	loss_disc=4.440, loss_gen=2.694, loss_fm=0.573,loss_mel=26.478, loss_kl=9.000
2023-12-01 08:47:16,894	SLILY	INFO	Stop Button was pressed. The program is closed.
2023-12-01 08:47:17,082	SLILY	INFO	saving final ckpt:Success.
2023-12-01 08:56:38,277	SLILY	INFO	{'train': {'log_interval': 200, 'seed': 1234, 'epochs': 20000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 16, 'fp16_run': True, 'lr_decay': 0.999875, 'segment_size': 12800, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'max_wav_value': 32768.0, 'sampling_rate': 40000, 'filter_length': 2048, 'hop_length': 400, 'win_length': 2048, 'n_mel_channels': 125, 'mel_fmin': 0.0, 'mel_fmax': None, 'training_files': './logs\\SLILY/filelist.txt'}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [10, 10, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'use_spectral_norm': False, 'gin_channels': 256, 'spk_embed_dim': 109}, 'model_dir': './logs\\SLILY', 'experiment_dir': './logs\\SLILY', 'save_every_epoch': 50, 'name': 'SLILY', 'total_epoch': 10000, 'pretrainG': 'pretrained_v2/f0G40k.pth', 'pretrainD': 'pretrained_v2/f0D40k.pth', 'version': 'v2', 'gpus': '0', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 1, 'save_every_weights': '1', 'if_cache_data_in_gpu': 0}
2023-12-01 08:56:39,604	SLILY	INFO	loaded pretrained pretrained_v2/f0G40k.pth
2023-12-01 08:56:39,774	SLILY	INFO	loaded pretrained pretrained_v2/f0D40k.pth
2023-12-01 08:56:54,403	SLILY	INFO	Train Epoch: 1 [0%]
2023-12-01 08:56:54,403	SLILY	INFO	[0, 0.0001]
2023-12-01 08:56:54,404	SLILY	INFO	loss_disc=4.437, loss_gen=2.697, loss_fm=0.575,loss_mel=26.523, loss_kl=9.000
2023-12-01 08:56:55,356	SLILY	INFO	Stop Button was pressed. The program is closed.
2023-12-01 08:56:55,460	SLILY	INFO	saving final ckpt:Success.
